<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-3A4C4UN" ignored="0" name="Test" tests="1" failures="1" timestamp="2020-06-27T18:04:22 CEST" time="0.531" errors="0">
  <testcase name="getUrl" time="0.018" classname="saucedemofail.SauceDemoTest">
    <failure type="java.lang.AssertionError" message="Incorrect url expected [https://www.saucedemo.co21m/] but found [https://www.saucedemo.com/]">
      <![CDATA[java.lang.AssertionError: Incorrect url expected [https://www.saucedemo.co21m/] but found [https://www.saucedemo.com/]
at saucedemofail.SauceDemoTest.getUrl(SauceDemoTest.java:37)
at java.util.ArrayList.forEach(Unknown Source)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- getUrl -->
</testsuite> <!-- Test -->
